File: 8KYU/Count of Positive Sum of Negatives/solution.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class Kata
{
    public static int[] CountPositivesSumNegatives(int[] input)
    {
      if (input is null || input.Length == 0)
        return new int[0];
      var positiveCount = 0;
      var negativeSum = 0;
      foreach(var n in input)
        if(n > 0)
          positiveCount ++;
        else
          negativeSum += n;
      return new[]{positiveCount, negativeSum};
    }
}

File: 8KYU/Count of Positive Sum of Negatives/solution.rs
fn count_positives_sum_negatives(input: Vec<i32>) -> Vec<i32> {
    if input.len() == 0 {
        return vec![];
    } else {
        let mut pos = 0;
        let mut neg_sum = 0;
        
        for x in input {
            if x > 0 {
                pos = pos + 1;
            } else {
                neg_sum = neg_sum + x;
            }
        }
        return vec![pos, neg_sum];
    }
}